classes/jbook.lua

```diff
@@ -29,7 +29,7 @@ function jbook:init()
   return book.init(self)
 end
 
-function jbook:registerCommands()
+jbook.registerCommands = function(_)
```

Comment:
I just looked over the other classes and we have a 50/50 split on usage. This will get fixed naturally when I refactor classes using Penlight instead of stdlib (which is a doozy to do without breaking stuff). I'm going to give this a pass for now.